tests/test_fetch.py

```diff
@@ -287,3 +288,12 @@ def test_query_caching(self):
 
         # reset cache directory state (will fail if purge was unsuccessful)
         os.rmdir(os.path.expanduser('~/dj_query_cache'))
+
+    def test_fetch_group_by(self):
+        # https://github.com/datajoint/datajoint-python/issues/914
+
+        try:
```

Comment:
This `try` ... `except` does not appear to be necessary. Just letting the exception raise will have the same effect.